src/routes/(app)/strafbakken/+page.svelte

```diff
@@ -1,15 +1,131 @@
-<script>
+<script lang="ts">
+  import type { PageData } from "../$types";
+  import { Plus, Minus } from "svelte-heros-v2";
+  import Modal from "./Modal.svelte";
+  import { openModal } from "svelte-modals";
+  import type { User } from "@prisma/client";
 
+  import { Modals, closeModal } from "svelte-modals";
+
+  interface sb extends User {
+    _count: {
+      StrafbakReceived: number;
+    };
+  }
+
+  interface sbPageData extends PageData {
+    strafbakken: sb[];
+  }
+
+  export let data: sbPageData;
+
+  const trekBak = (id: number, index: number) => {
+    changeCount(index, -1);
+    fetch("/strafbakken", {
+      method: "DELETE",
+      body: JSON.stringify({
+        user: id,
+      }),
+    }).catch(console.error);
+  };
+
+  const changeCount = (index: number, n: number) => {
+    data.strafbakken[index]._count.StrafbakReceived += n;
+  };
 </script>
 
-<div>
-  <h1>
-    Success Naut
-  </h1>
-</div>
+<main>
+  <Modals>
+    <!-- svelte-ignore a11y-click-events-have-key-events -->
```

Comment:
Accessibility is voor sukkels

Reply:
dit is gewoon een copy pasta joh geen idee